index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <title>JS Quiz</title>
+    </head>
+    <body style="text-align: center; vertical-align: middle; margin-top: 15%;">
+        <h1>Input size of tree to create:</h1>
+        <form id="form" action="index.html" method="post">
```

Comment:
Forms will automatically take care of an `Enter` press. For this exercise we want you to handle that on your own. I know you have a check for it, but the form is probably overriding it. The only way to be sure is to just get rid of the `<form>`.